stable/harbor/questions.yaml

```diff
@@ -0,0 +1,577 @@
+labels:
+  io.cattle.role: project # options are cluster/project
+questions:
+# Global Options
+- variable: harborAdminPassword
+  required: true
+  description: "The initial password of Harbor admin. Change it from portal after launching Harbor"
+  type: password
+  label: The Initial Password of Harbor Admin
+  group: "Global Options"
+- variable: persistence.enabled
+  default: true
+  required: true
+  description: "Enable persistent storage for Harbor components"
+  type: boolean
+  label: Enable Persistent Storage
+  group: "Global Options"
+# Harbor Expose Service
+- variable: expose.type
+  required: true
+  default: "ingress"
+  description: "Set the way how to expose the service"
+  type: enum
+  label: Harbor Proxy Service Type
+  group: "Expose Service Options"
+  options:
+  - "ingress"
+  - "loadBalancer"
+  - "nodePort"
+  - "clusterIP"
+- variable: expose.ingress.hosts.core
+  required: true
+  default: "harbor.domain"
+  description: "The host of Harbor core service in ingress rule, e.g, harbor.domain"
+  type: string
+  label: Ingress Hostname of Harbor
+  group: "Expose Service Options"
+  show_if: "expose.type=ingress"
+- variable: expose.ingress.hosts.notary
+  required: true
```

Comment:
I'm wondering if this should be strictly required, as notary itself could be disabled...

Reply:
You're right. We can hide this if notary is disabled, but keep it required when it does show up.